portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,17 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  const quotes =
+      ['Yeah Science!', 'SPS is lit', 'I am not throwing away my shot', 'I’m just a simple man trying to make my way in the universe'];
```

Comment:
One thing to consider doing here is moving the quotes array outside of the addRandomQuote function. That way you don't re-create it each time the function runs.

Reply:
I was listening to the soundtrack while working on this, its a great musical!